### DISCUSSION TITLE
How do you make a field type string or array of string?

### DISCUSSION THREAD

**Discussion by JakeCoxon** [dynamoose/dynamoose#1207] [Q&A]
I'm looking to add a field type of either String or Array of String but I can't seem to get it working, but i may have just misunderstood the documentation

```
export const AnswerField = dynamoose.model(
  'answersTable',
  {
    user: String,
    fieldName: String,
    value: [String, { type: Array, schema: [String] }]
  },
  { create: false }
)
```

This one doesn't typecheck and with ts-ignore it gives me an error

```
TypeError: node.forEach is not a function
    at /node_modules/dynamoose/lib/Document.ts:277:10
```

Is there another syntax I have to use? or is it not possible?

**Comment by fishcharlie** [comment]
@JakeCoxon I feel like this should work. Wanna create an issue as a bug so I can look into this further?

**Reply by JakeCoxon** [reply]
hi @fishcharlie I tried digging into the code to resolve it myself but I couldn't. I ended up removing dynamoose from my code base so I'm sorry I can't help anymore
